### ISSUE TITLE
A question regarding how the grid search over hyperparameters was performed

### ISSUE THREAD

**Issue by yjlolo** [JunwenBai/C-DSVAE#4]
Thank you for sharing the code!

I have a question about the grid search. In Section 4.3 from the paper, it is written

> Models are trained until convergence and the one with the best validation accuracy would be chosen for testing.

I wonder if the accuracy refers to the metric ACC. For example, were the TIMIT hyperparameters chosen based on the best speaker identification accuracy on the validation set?

**Comment by JunwenBai** [comment]
yeah, generally the best metric scores on the validation set.
